src/extractors/gitmojiextractor.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copied and changed from nerdextractor.py in https://github.com/fdw/rofimoji
+"""
+
+
+import html
+from pathlib import Path
+import json
+
+import requests
+
+from characterfactory import Character
+
+
+class GitmojiExtractor(object):
+    def __init__(self: 'GitmojiExtractor'):
+        self.icons = []
+
+    def fetch_icons(self: 'GitmojiExtractor'):
+        print("Downloading list of gitmojis")
+
+        response = requests.get(
+            "https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json",
+            timeout=60
+        )
+        """
+        {
+            'emoji': '🎨',
+            'entity': '&#x1f3a8;',
+            'code': ':art:',
+            'description': 'Improve structure / format of the code.',
+            'name': 'art',
+            'semver': None
+        }
+        """
+        for char in response.json()["gitmojis"]:
+            icon = ord(char["emoji"][0])
```

Comment:
Are we sure there is only ever one character?

Reply:
I am using only the first character of the string. No issues with it so far.